### ISSUE TITLE
Accessibility issue -- no alt= attributes on vignette plots

### ISSUE THREAD

**Issue by MichaelChirico** [rvlenth/emmeans#389]
The HTML vignettes on CRAN have some images lacking `alt=` attributes.

 - [ ] interactions.html:144
 - [ ] messy-data.html:229
 - [ ] sophisticated.html:613,639

(NB: these line numbers in the HTML output, not necessarily the same line as in the .Rmd)

See https://www.rstudio.com/blog/knitr-fig-alt/ for more about how to add them and why. Thanks!

**Comment by rvlenth** [comment]
Hello,

Thanks for your bringing this to my attention. It is a good suggestion that I had not been aware of, and I will start working on it. This enhancement is needed not only for the three vignettes you mention, but for about 10 vignettes in all.

A related issue is whether to add a `title` tag rather than, or in addition to, the `alt=` tag. From what I can tell, in most browsers, `title=` text is displayed to all users in a little pop-up, whereas `alt=` text typically does not pop up and is displayed only until the figure has loaded. I guess this is moot, because as far as I can tell, there is no `fig.title` tag in **knitr**, only `fig.alt`; so it would be hard to add `title=` tags. Nonetheless, am I correct that contents of `title=` tags *would* be available to screen readers?

I will tag this issue when I upload related updates.

**Comment by rvlenth** [comment]
I think this has been addressed, so am closing.
